crates/bevy_math/src/primitives/mod.rs

```diff
@@ -0,0 +1,13 @@
+//! This module defines primitive shapes.
+//! The origin of all primitives is at 0,0(,0) unless stated otherwise
```

Comment:
```suggestion
//! The origin of all primitives is at 0,0(,0) unless stated otherwise.
```

There's a lot of other docs sentences missing a final `.`.